src/transformers/cli/serve.py

```diff
@@ -720,51 +717,45 @@ def chunk_to_sse_element(chunk: ChatCompletionChunk | BaseModel) -> str:
         """
         return f"data: {chunk.model_dump_json(exclude_none=True)}\n\n"
 
-    @functools.cache
-    def get_gen_models(self) -> list[dict[str, any]]:
+    @staticmethod
+    def get_gen_models(cache_dir: Optional[str] = None) -> list[dict[str, any]]:
         """
-        This is by no means a limit to which models may be instantiated with `transformers serve`: any chat-based
-        model working with generate can work.
-
-        This is a limited list of models to ensure we have a discoverable /v1/models endpoint for third-party
-        integrations.
+        List generative models in the cache.
```

Comment:
more a list of generative language models? (or maybe generative models implies "text generation" in `transformers`?)